apps/editorial_features/components/eoy/templates/index.jade

```diff
@@ -9,19 +9,20 @@ block head
 
 block body
   - var superArticle = article
-  include ../../../../../components/article/templates/super_article_sticky_header.jade
+  .eoy-feature__sa-header
+    include ../../../../../components/article/templates/super_article_sticky_header.jade
 
   .eoy-feature
-    video(class='video' autoplay=true, controls=false, loop=true, muted=true, playsinline=true).eoy-feature__background.active
+    video(class='video' autoplay=true, controls=false, loop=true, muted=true, playsinline=true poster=curation.get('placeholder')).eoy-feature__background.active
```

Comment:
Hmm yes good idea, but we'll want to do different images for those probably. I'll make a note in the issue tracker and mark it as "nice to have". thank you!